docs/community/tutorials-contest/2022/local-subnet-development/README.md

```diff
@@ -0,0 +1,358 @@
+# Set up your development environment for local subnet development
+
+If you are not familiar with subnets, virtual machines or similar terminology you can refer to [Subnet Overview](https://docs.avax.network/subnets).
+
+## Introduction
+
+This tutorial's goal is to deploy and start a basic subnet in your local machine. So that you can interact with the subnet using [Remix](https://remix.ethereum.org/) and [Hardhat](https://hardhat.org/). In this tutorial we will be using [avalanche-cli](https://github.com/ava-labs/avalanche-cli) to create and deploy the subnet. If you ever encounter an error, refer to [Troubleshoot Common Issues](#troubleshoot-common-issues) section.
+
+:::info avalanche-cli is in beta version. So it might get updated fairly frequently. It is best to refer to the latest version from its [github page](https://github.com/ava-labs/avalanche-cli). :::
```

Comment:
separate new line after `:::info` and before `:::` by following https://docusaurus.io/docs/markdown-features/admonitions

Please verify the text is displayed as expected, not ill-formated.